protocol-units/execution/aptos/util/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "monza-execution-util"
+name = "movement-execution-util"
```

Comment:
Again, if we're talking about extracting generalized `aptos` executors, I would title these as such.